libs/cart/src/drivers/injection-tokens/cart-driver.token.ts

```diff
@@ -1,3 +1,4 @@
 import { InjectionToken } from '@angular/core';
+import { DaffCartServiceInterface } from '../interfaces/cart-service.interface';
 
-export const DaffCartDriver = new InjectionToken('DaffCartDriver');
+export const DaffCartDriver = new InjectionToken<DaffCartServiceInterface<any>>('DaffCartDriver');
```

Comment:
I don't think this actually enforces anything. I've tested it by injecting something else, and the build still doesn't seem to fail. Do you think we should keep it or remove it?

Reply:
I vote keep as it will eventually do something in a later version of Angular.

Reply:
ok sure